vms/spdagvm/vm.go

```diff
@@ -328,10 +330,19 @@ func (vm *VM) Send(amount uint64, assetID, toAddrStr string, fromPKs []string) (
 		if err != nil {
 			return "", err
 		}
-		// Parse the crpyo.PrivateKey repr. to a crypto.PrivateKeySECP256K1R
+		// Parse the crypto.PrivateKey repr. to a crypto.PrivateKeySECP256K1R
 		keychain.Add(pk.(*crypto.PrivateKeySECP256K1R))
 	}
 
+	toAddrBal, err := vm.GetBalance(toAddrStr, "")
```

Comment:
Thinking this balance check should be moved to builder.NewTxFromUTXOs to cover the wallet's Send function as well.